rmf_task_ros2/src/rmf_task_ros2/Dispatcher.cpp

```diff
@@ -294,9 +300,15 @@ class Dispatcher::Implementation
         this->handle_dispatch_ack(*msg);
       });
 
+    task_state_update_pub = node->create_publisher<TaskStateUpdateMsg>(
+      rmf_task_ros2::TaskStateUpdateTopicName,
+      rclcpp::ServicesQoS().keep_last(10).reliable().transient_local());
```

Comment:
Is there a reason why this is using a different QoS from https://github.com/open-rmf/rmf_ros2/pull/383/files#diff-979ff9700f77b8e1260de3301125242adb32556ba5092fcddcc39e91baa09a2dR195-R196?

Reply:
thanks for spotting that, I've corrected them to be the same https://github.com/open-rmf/rmf_ros2/pull/383/commits/07b9c6abc23200c30123bf681c2ceedc62458dba